client.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
```

Comment:
Does this replace the shell script? Maybe update the README.md to reflect that and delete the shell script?

Reply:
Or is this for debugging commands rather than logging? Are the debug logs and commands on different ports?

Reply:
Client.py is our method of allowing Node-RED to communicate bluedonkey.py, and is intended as a permanent addition to the workflow.